api/src/main/java/org/apache/cloudstack/api/command/admin/network/UpdateNetworkOfferingCmd.java

```diff
@@ -55,7 +55,7 @@ public class UpdateNetworkOfferingCmd extends BaseCmd {
     private String displayText;
 
     @Parameter(name = ApiConstants.AVAILABILITY, type = CommandType.STRING, description = "the availability of network offering."
-            + " Default value is Required for Guest Virtual network offering; Optional for Guest Direct network offering")
+            + " The default value is Required for Guest Virtual network offering, Optional for Guest Direct network offering. Only one network offering can have the value Required ")
```

Comment:
It was not changed in this PR but I don ´t understand `Required for Guest Virtual network offering, Optional for Guest Direct network offering`. it makes the whole thing strange and incomprehensible. 
Also I don't think it will make sense to users that the default is different (as discribed) for the create vs the update API. I am not sure if we need to change the code or the doc string, but more work is needed, i think.

Reply:
agree
actually the default value is null in  [UpdateNetworkOfferingCmd.java](https://github.com/apache/cloudstack/pull/7545/files#diff-262c2af825c2e0b2c589174a93de44fe0d332ad36c321634f0f1f7ca69bc8fd5)